src/parameterizations/lateral/MOM_internal_tides.F90

```diff
@@ -1272,110 +1272,111 @@ subroutine propagate_corner_spread(En, energized_wedge, NAngle, speed, dt, G, CS
           a2 = (yCrn - yW)*(0.5*(xCrn + xW))
           a3 = (yW - yNW)*(0.5*(xW + xNW))
           a4 = (yNW - yN)*(0.5*(xNW + xN))
-          aW = a1 + a2 + a3 + a4
+          aW = a1 + (a2 + (a3 + a4))
           ! southwest area
           a1 = (yCrn - yS)*(0.5*(xCrn + xS))
           a2 = (yS - ySW)*(0.5*(xS + xSW))
           a3 = (ySW - yW)*(0.5*(xSW + xW))
           a4 = (yW - yCrn)*(0.5*(xW + xCrn))
-          aSW = a1 + a2 + a3 + a4
+          aSW = a1 + (a2 + (a3 + a4))
```

Comment:
This expression for aSW is not rotationally invariant.  If I have done the algebra right, a mathematically equivalent but rotationally invariant form would be `aSW = 0.5 * ((yCrn - ySW) * (xS - xW) + (yW - yS) * (xCrn - xSW))`.

A similar comment would apply to all of the other areas.